### ISSUE TITLE
"Zbay light" supports public channels 

### ISSUE THREAD

**Issue by holmesworcester** [ZbayApp/zbay#231]
Depends on #221

* Parse the "channel of channels" correctly
* User can join a public channel
* User can publish a channel

**Comment by holmesworcester** [comment]
On "user can join a channel" our requirement is that user will see new messages, and a alert (see figma) for the fact that they must restart to see older messages.

On next restart we will rescan and all old messages will be visible in all added channels.

**Comment by holmesworcester** [comment]
Link to design https://www.figma.com/file/0j7Nna9zWmfOSNmRmQK1Uh/Zbay-Design?node-id=1934%3A0

**Comment by NorbertBodziony** [comment]
Joining public channel will require spawning new wallet-cli worker and import key there.

**Comment by NorbertBodziony** [comment]
I think these designs are not in scope of this task. User right now can join public channels and send/receive messages right away. Publishing channels should be completely different task since it will require creating new channels. I will create one task for that.

**Comment by holmesworcester** [comment]
great
